status_test.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	config "github.com/remotemobprogramming/mob/v4/configuration"
+	"strconv"
+	"testing"
+)
+
+func TestExecuteKicksOffStatus(t *testing.T) {
+	output, _ := setup(t)
+
+	execute("status", []string{}, config.GetDefaultConfiguration())
+
+	assertOutputContains(t, output, "you are on base branch 'master'")
+}
+
+func TestStatusMobProgramming(t *testing.T) {
+	output, configuration := setup(t)
+	start(configuration)
+
+	status(configuration)
+
+	assertOutputContains(t, output, "you are on wip branch mob-session")
+}
+
+func TestStatusWithMoreThan5LinesOfLog(t *testing.T) {
+	output, configuration := setup(t)
+	configuration.NextStay = true
+	start(configuration)
+
+	for i := 0; i < 6; i++ {
+		createFile(t, "test"+strconv.Itoa(i)+".txt", "contentIrrelevant")
+		next(configuration)
+	}
+
+	status(configuration)
+	// 6 wip commits + 1 start commit
+	assertOutputContains(t, output, "wip branch 'mob-session' contains 7 commits.")
+}
+
+func TestStatusDetectsWipBranches(t *testing.T) {
+	output, configuration := setup(t)
+	start(configuration)
```

Comment:
I would like to adjust the test that it does not depend on the empty commit when doing mob start. As I would like to remove this empty commit again.

Reply:
ok, great!